csrc/fusion_segmenter.cpp

```diff
@@ -1424,7 +1424,7 @@ void SegmentedFusion::print() const {
   std::cout << this << "\n";
 }
 
-std::string toString(SegmentedFusion* segmented_fusion) {
+std::string toString(const SegmentedFusion* segmented_fusion) {
```

Comment:
Looks like this function can also be marked const~

Reply:
This is not a member function, so it cannot be made `const` given that it doesn't own any state to modify.